demisto_sdk/commands/validate/tests/IN_validators_test.py

```diff
@@ -1909,3 +1912,43 @@ def test_IsContainingMultipleDefaultArgsValidator_is_valid(
             for result, expected_msg in zip(results, expected_msgs)
         ]
     )
+
+
+def test_IntegrationDisplayNameVersionedCorrectlyValidator_is_valid():
+    """
+    Given:
+     - 1 integration with valid versioned display-name
+     - 1 integration with invalid versioned display-name
+
+    When:
+     - Running the IntegrationDisplayNameVersionedCorrectlyValidator validator
+
+    Then:
+     - make sure the integration with the invalid version fails on the validation
+    """
+    content_items = [
+        create_integration_object(paths=["display"], values=["test v2"]),
+        create_integration_object(paths=["display"], values=["test V3"]),
+    ]
+
+    results = IntegrationDisplayNameVersionedCorrectlyValidator().is_valid(
+        content_items
+    )
+    assert len(results) == 1
+    assert results[0].content_object.display_name == "test V3"
```

Comment:
let's assert the received message instead.

Reply:
as said previously i am not a huge fan of asserting expected messages as it can vary all the time and it just breaks the test for now reason, eventually I want to test here that the display_name was actually changed as expected.